docs/scripts/build-component-props.mjs

```diff
@@ -0,0 +1,57 @@
+import fs from 'fs-extra';
+import path from 'path';
+import { dirname } from 'path';
```

Comment:
Imported twice. Also when importing node modules, we should use node:path for security :)